model/src/wav_grdout.F90

```diff
@@ -225,7 +225,8 @@ subroutine initialize_gridout
     varatts( "USP  ", "USSPY     ", "Partitioned surface Stokes drift y              ", "m s-1     ", "p ", .false.) , &
     varatts( "TWC  ", "TAUOCX    ", "Total wave to ocean stress x                    ", "Pa        ", "  ", .false.) , &
     varatts( "TWC  ", "TAUOCY    ", "Total wave to ocean stress y                    ", "Pa        ", "  ", .false.) , &
-    varatts( "LAN  ", "LANGMT    ", "Turbulent Langmuir number (La_t)                ", "nd        ", "  ", .false.)   &
+    varatts( "USSH ", "USSHX     ", "Surface layer averaged Stokes drift x           ", "m s-1     ", "  ", .false.) , &
+    varatts( "USSH ", "USSHY     ", "Surface layer averaged Stokes drift y           ", "m s-1     ", "  ", .false.) , &
```

Comment:
You need to remove the comma at the end here (before ampersand).

Reply:
Done